packages/suite/src/services/github.ts

```diff
@@ -17,4 +19,49 @@ export const getReleaseNotes = async (version?: string) => {
     return release;
 };
 
+export const openGithubIssue = async () => {
+    const url = new URL(`${RELEASE_URL}/issues/new`);
+
+    let type = 'web';
+    if (isDesktop()) {
+        type = 'desktop app';
+    }
+
+    const body = `
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Steps to reproduce:**
+1. a
+2. b
+3. c
+
+**Info:**
+ - Suite Version: ${type} ${process.env.VERSION} (${process.env.COMMITHASH})
+ - Browser: ${getUserAgent()}
+ - Screen: ${getScreenWidth()}x${getScreenHeight()}
+ - OS: ${navigator.platform}
+ - Firmware Version: TODO
+
+**Expected result:**
+A clear and concise description of what you expected to happen.
+
+**Actual result:**
+A clear and concise description of what actually happens.
+
+**Screenshots:**
+insert here
+
+**Note(s):**
+Add any other context about the problem here.
+
+**More info**
+<details>TODO</details>
+`;
+
+    url.searchParams.set('body', body);
+
+    await window.open(url.toString());
```

Comment:
there is nothing to `await` here :] so this could be removed (and `async` keyword too)

Reply:
926c3ac637d1d3c9bb07f0d669b6ccb068cda346